client/driver/docker.go

```diff
@@ -1119,6 +1130,12 @@ func (d *DockerDriver) createContainerConfig(ctx *ExecContext, task *structs.Tas
 		VolumeDriver: driverConfig.VolumeDriver,
 	}
 
+	// Calculate CPU Quota
+	if driverConfig.CPUHardLimit {
+		percentTicks := float64(task.Resources.CPU) / shelpers.TotalTicksAvailable()
```

Comment:
You should use `d.node.Resources.CPU` instead as we don't always properly detect the available CPU so some users have to override it.